docs/sockets.md

```diff
@@ -0,0 +1,98 @@
+# Web Socket Documentation
+Find the code for all socket endpoints here: [server/sockets.js](https://github.com/Yonava/RhymeTime/blob/main/server/sockets.js)
+
+Our project utilizes tooling built by [Socket.io](https://socket.io/docs/v4/) for implementing sockets that use the websocket protocol integrated with http for automatic reconnection
+
+```npm install socket.io```
+[npm](https://www.npmjs.com/package/socket.io)
+
+Socket endpoints are split into 2 categories: <a href="#connection">Connection Endpoints</a> & <a href="#gameplay">Gameplay Endpoints</a>
+<br><br>
+Connection endpoints are endpoints that exist solely to facilitate the function of connecting clients together for the purpose of making gameplay possible
+<br><br>
+Gameplay endpoints, in the other hand, are endpoints that are used by each client during gameplay to send data like new prompts, which prompt another player votes for etc.
+
+<div id="connection"></div>
+
+## Connection Endpoints
```

Comment:
Can we add the type of connection to communicate with each? (GET, PUT, POST, etc)

Reply:
That is not applicable for these endpoints